GCICAP.lua

```diff
@@ -522,6 +528,7 @@ do
                           command = "local group = ...\
                           local flight = gcicap.Flight.getFlight(group)\
                           if flight then\
+                            flight.give_up = true\
```

Comment:
Found a bug with this and GCI flights. This triggers really quickly and I'm not sure why yet. Possibly because the flights start on the ground - maybe they need two points at the target, a move and then the engage. I'd hold off merging until I address it, it makes GCI flights almost completely broken.